src/Nox.ClientApi.Tests/Tests/Controllers/WorkplacesControllerTests.cs

```diff
@@ -557,35 +559,93 @@ public async Task Get_LocalizedValueNotFound_ShouldReturnDefaultValue()
             localizedWorkplaces![0].Description.Should().EndWith("]");
         }
 
-        // TODO: implement once create command works
-        //[Fact]
-        //public async Task Get_LocalizedValue_ShouldReturnLocalizationValue()
-        //{
-        //    var nameFixture = _fixture.Create<string>();
-
-        //    // Arrange
-        //    var createDto = new WorkplaceCreateDto
-        //    {
-        //        Name = nameFixture,
-        //        Description = _fixture.Create<string>(),
-        //    };
-
-        //    await PostAsync<WorkplaceCreateDto, WorkplaceDto>(Endpoints.WorkplacesUrl, createDto);
-
-        //    var headers = new Dictionary<string, IEnumerable<string>>()
-        //    {
-        //        { "Accept-Language", new List<string> { $"fr-CH, fr;q=0.9, en;q=0.8, de;q=0.7, *;q=0.5" } }
-        //    };
-
-        //    // Act
-        //    var localizedWorkplaces = (await GetODataCollectionResponseAsync<IEnumerable<WorkplaceDto>>($"{Endpoints.WorkplacesUrl}", headers))?.ToList();
-
-        //    // Assert
-        //    localizedWorkplaces.Should().NotBeNull();
-        //    localizedWorkplaces.Should().HaveCount(1);
-        //    localizedWorkplaces![0].Description.Should().NotBeNull();
-        //    localizedWorkplaces![0].Description.Should().StartWith("[");
-        //    localizedWorkplaces![0].Description.Should().EndWith("]");
-        //}
+        [Fact]
+        public async Task GetById_LocalizedValueNotFound_ShouldReturnDefaultValue()
+        {
+            var nameFixture = _fixture.Create<string>();
+
+            // Arrange
+            var createDto = new WorkplaceCreateDto
+            {
+                Name = nameFixture,
+                Description = _fixture.Create<string>(),
```

Comment:
Can we have a hardcoded Description here, and check the exact value of the Description, I can always create a workplace with a Description=[Test] and this test would pass.